ios/Podfile

```diff
@@ -1,41 +0,0 @@
-# Uncomment this line to define a global platform for your project
```

Comment:
Same questions as above, why does this change delete a bunch of iOS related files?